### PR TITLE
Draft release notes for 1.16.1

### PR DESCRIPTION
**Pull request by AtnNn**
@danielmewes, @mglukhovsky could you please review?

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# Release 1.16.1 (Stand By Me)
+
+Released on 2015-02-12
+
+Bug fix update.
+
+* Now write to the log every time a file is deleted (#1780)
```
Comment (danielmewes) [inline NOTES.md]: I'd change this to "Write a message to the log every time a file is deleted"

File: NOTES.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+* Fixed `rethinkdb dump` and other backup scripts to correctly detect the server version (#3706)
+* Changed the output of `rethinkdb dump` to clarify that indexes are being saved (#3708)
+* Fixed memory consumption with newer versions of boost (#3712)
```
Comment (danielmewes) [inline NOTES.md]: We should probably mention that this affected our OS X dmg.
How about
"Fixed unbounded memory consumption when using the official OS X package or when building with boost version 1.56.0 or higher"

File: NOTES.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+* Fixed the `written_docs_total` field of `rethinkdb.stats` (#3713)
+* Fixed a bug that caused the web UI to hang when creating indexes (#3718)
+* Fixed how secondary index status was being reported (#3721)
```
Comment (danielmewes) [inline NOTES.md]: I would merge this with 3718 into
"Fixed a bug that caused the web UI to hang when creating or deleting indexes (#3717, #3721)"

File: NOTES.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+* Fixed rounding of document counts in the web UI (#3722)
+* Fixed a bug that broke the `-f` flag of `rethinkdb import` (#3728)
+* Fixed a bug that caused information to go missing in the web UI (#3729)
```
Comment (danielmewes) [inline NOTES.md]: I'd prefer "Fixed a bug that stopped the web UI from loading all information properly"

**Comment by danielmewes** [comment]
Added comments. Everything else looks good to me.

File: NOTES.md (hunk 4 above)
Comment (mglukhovsky) [inline NOTES.md]: I'd say "Fixed a bug that prevented the web UI from loading data properly"

File: NOTES.md [hunk 5]
```diff
@@ -1,3 +1,24 @@
...
+* Fixed a bug that caused RethinkDB to refuse to link with jemalloc dynamically (#3741)
+* Fixed an uncaught exception in the handling or `r.js` (#3745)
```
Comment (mglukhovsky) [inline NOTES.md]: or -> of

**Comment by mglukhovsky** [comment]
I agree with @danielmewes's fixes, added my own comments.

**Comment by AtnNn** [comment]
Thanks. I've incorporated your suggestions.

**Comment by mglukhovsky** [comment]
Looks good to me.

**Comment by AtnNn** [comment]
Merged in c9e6a8c

**Pull request closed without merging**